src/main/java/com/powsybl/openloadflow/network/util/ActivePowerDistribution.java

```diff
@@ -86,8 +86,8 @@ public Result run(LfGenerator referenceGenerator, Collection<LfBus> buses, doubl
             iteration++;
         }
 
-        final boolean movedBuses = initialP.entrySet().stream()
-                .anyMatch(e -> Math.abs(e.getKey().getTargetP() - e.getValue()) > P_RESIDUE_EPS);
+        final boolean movedBuses = Math.abs(initialP.entrySet().stream()
+                .mapToDouble(e -> e.getKey().getTargetP() - e.getValue()).sum()) > P_RESIDUE_EPS;
```

Comment:
Isn't better to sum the abs values instead of doing the abs value of the sum ?

Reply:
in today's distribution all elements go always in the same direction, therefore we can spare a few cpu cycles with a single Math.abs
But maybe we should indeed put abs on each element, no strong opinion here ...

Reply:
Oh yes that's true. No strong opinion neither, these cpu cycles may not be perceptible, but it can stay as it is.

Reply:
in the end I change to what you suggested, imagine in the future we do fancy distribution on both loads and generators...